test/test_migrate.rb

```diff
@@ -44,7 +44,7 @@ def load_script(**args)
   end  
 end
 
-unless ENV['TRAVIS']
+unless ENV['CI']
```

Comment:
This environment is in the default environment variables for the travis CI and github actions.